liquibase-core/src/main/java/liquibase/util/FileUtil.java

```diff
@@ -43,23 +43,13 @@ public static void write(String contents, File file, boolean append) throws IOEx
     }
 
     public static String getFileNotFoundMessage(String physicalChangeLogLocation) {
-        //
-        // Check for any prefix which is not file:
-        // Take this to indicate a full path
-        //
-        if (physicalChangeLogLocation.matches("^\\w\\w+:.*") && ! physicalChangeLogLocation.startsWith("file:")) {
-            return "ERROR: The file '" + physicalChangeLogLocation + "' was not found." + System.lineSeparator() +
-                    "The file property cannot be configured with a fully qualified path, but must be a relative path on the property," + System.lineSeparator() +
```

Comment:
Where did this message about the fully qualified path go? I am not seeing it in the PR but I'm willing to bet I am just missing it. 

CC @arturobernalg and @MalloD12

Reply:
Hi @XDelphiGrl, I think I missed this when reviewing this PR. I searched for it and haven't seen it elsewhere. Unless @arturobernalg has a reason for it, I'll revert this message back to this method.

Reply:
That message should be there.

Reply:
Cool, thank you for the confirmation. I'll add it back. Thank you @arturobernalg and thank you @XDelphiGrl for catching this up.